src/HospitalRun.tsx

```diff
@@ -75,6 +76,15 @@ const HospitalRun = () => {
                     path="/appointments/new"
                     component={NewAppointment}
                   />
+                  <PrivateRoute
+                    isAuthenticated={
+                      permissions.includes(Permissions.WriteAppointments) &&
+                      permissions.includes(Permissions.ReadAppointments)
+                    }
+                    exact
+                    path="/appointments/edit/:id"
+                    component={EditAppointment}
```

Comment:
Could we add some tests around this in `HospitalRun.test.tsx`. Just looking for tests around the security and then making sure the component renders when the route is accessed.

Reply:
Related to this, do you think it should require any Patient permissions for this route?

Reply:
Probably since there is patient data that can be seen... 

We probably need a `ReadPatient` and `ReadPatients` permission. New Appointments and Edit Appointments should require `ReadPatients` maybe?

I think that is outside the scope of this issue though.

Reply:
Ok, added tests.